### PR TITLE
modbus:serial added "SerialEncodings" selectability 

### PR DESCRIPTION
**Pull request by nickma82**
such that one is able to setup the serial encoding (e.g. ascii, rtu, bin)
along with the other "connection" port settings via the openhab.cfg

First pull request since.. quite a time via github, be nice ;)

### CODE REVIEW

File: bundles/binding/org.openhab.binding.modbus/src/main/java/net/wimpi/modbus/net/SerialConnection.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ public void open() throws Exception {
           CommPortIdentifier.getPortIdentifier(m_Parameters.getPortName());
     } catch (NoSuchPortException e) {
       if(Modbus.debug) System.out.println(e.getMessage());
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.modbus/src/main/java/net/wimpi/modbus/net/SerialConnection.java]: please replace the System.out with proper Logging

File: bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusBinding.java [hunk 2]
```diff
@@ -106,21 +108,30 @@ protected void internalReceiveCommand(String itemName, Command command) {
...
+				newState = newState.equals(DecimalType.ZERO) ? OnOffType.OFF : OnOffType.ON;
+			}
+			if (( rawDataMultiplier != 1 ) && (config.getItem() instanceof NumberItem)) {
+				double tmpValue = (double)((DecimalType)newState).doubleValue() * rawDataMultiplier;
+				newState =  new DecimalType( String.valueOf(tmpValue) );
 			}
+
+			State currentState = config.getItemState();
+			if (! newState.equals(currentState))
+				eventPublisher.postUpdate(itemName, newState);
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/ModbusBinding.java]: please add curly braces (although this is just a One-Liner)

**Comment by teichsta** [comment]
Hi @nickma82 , thanks for this contribution. Please find my review comments inline. Thanks, Thomas E.-E.

File: bundles/binding/org.openhab.binding.modbus/src/main/java/net/wimpi/modbus/net/SerialConnection.java (hunk 1 above)
Comment (nickma82) [inline bundles/binding/org.openhab.binding.modbus/src/main/java/net/wimpi/modbus/net/SerialConnection.java]: Hi @teichsta , there's a huge load of System.out logging in the modbus binding, changing one seems like a drop in a bucket and I'm leaving for vacation today.

My offer: I'm going to change a bunch of them in a couple of weeks, when I'm back.

Comment (teichsta) [inline bundles/binding/org.openhab.binding.modbus/src/main/java/net/wimpi/modbus/net/SerialConnection.java]: Agreed :-) Would be great if you could send a new PR for that.

Enjoy your holidays!

Thanks, Thomas E.-E.

**Pull request merged**